presto-main/src/test/java/com/facebook/presto/operator/scalar/TestDateTimeFunctionsBase.java

```diff
@@ -756,6 +756,7 @@ public void testDateFormat()
         assertFunction("date_format(" + dateTimeLiteral + ", '%g')", VARCHAR, "g");
         assertFunction("date_format(" + dateTimeLiteral + ", '%4')", VARCHAR, "4");
         assertFunction("date_format(" + dateTimeLiteral + ", '%x %v')", VARCHAR, "2001 02");
+        assertFunction("date_format(" + dateTimeLiteral + ", '%Y年%m月%d日')", VARCHAR, "2001年01月09日");
```

Comment:
We don't usually use non-ascii code in our codebase. Could you convert it into unicode encoding?
```java
'%Y\u5e74%m\u6708%d\u65e5'

Reply:
Sure. Let me update.